projects/view/client/singleProject/singleProjectDetailed.css

```diff
@@ -0,0 +1,645 @@
+/*  SECTIONS  */
```

Comment:
This file contains many absolute units, i.e. `px` values. Convert most/all absolute units to relative units `em` or `rem`. Specifically, any rules that affect layout should be expressed in relative units.